src/cloud-api-adaptor/install/charts/peerpods/values.yaml

```diff
@@ -0,0 +1,69 @@
+# Default values for peerpods (CAA) chart
+
+# Namespace where peerpods resources will be deployed
+namespace: confidential-containers-system
+
+# CAA (Cloud API Adaptor) image
+image:
+  name: quay.io/confidential-containers/cloud-api-adaptor
+  # tag defaults to Chart.appVersion if not set
+  tag: ""
```

Comment:
Hi @beraldoleal ,

Note that we release two "variants" of the cloud-api-daptor:
* release: doesn't have libvirt-dev libraries
* dev: does have libvirt-dev libraries

The only provider that (I know of) uses the dev image is libvirt.

if the tag is based on `Chart.appVersion` then we will need to distribute two charts image (IIUC). Otherwise, we will need to use the `Values.provider` to figure out the proper tag.

Reply:
Sorry, can you clarify the context of this question please? During development we want it to be latest as we need to test against that, then during the release we either tag with dev (for libvirt & docker), or the release sha e.g. https://github.com/confidential-containers/cloud-api-adaptor/commit/0275f8b095db93ad3296a17b838cbcd6f810e0ae Are you suggesting that this approach doesn't work with charts and we can only have a single value?

Reply:
I will be honest that I've lost track of how `latest` works for all when the "release" doesn't, but maybe we need to crack open this and check that libvirt still requires the cgo bindings which I think is the reason for two images